src/utils/par.ts

```diff
@@ -60,22 +69,25 @@ export const makeParRequest =
         The key is matched by its kid */
     const signedJwtForPar = await new SignJWT(wiaCryptoContext)
       .setProtectedHeader({
+        typ: "jwk",
         kid: wiaPublicKey.kid,
       })
       .setPayload({
-        iss,
-        aud,
         jti: `${uuid.v4()}`,
-        client_assertion_type: assertionType,
-        authorization_details: authorizationDetails,
+        aud,
         response_type: "code",
-        redirect_uri: walletProviderBaseUrl,
-        state: `${uuid.v4()}`,
+        response_mode: responseMode,
         client_id: clientId,
-        code_challenge_method: codeChallengeMethod,
+        iss,
+        state: generateRandomAlphaNumericString(32),
         code_challenge: codeChallenge,
+        code_challenge_method: codeChallengeMethod,
+        authorization_details: authorizationDetails,
+        redirect_uri: redirectUri,
+        client_assertion_type: assertionType,
+        client_assertion: walletInstanceAttestation + "~" + signedWiaPoP,
       })
-      .setIssuedAt()
+      .setIssuedAt() //iat is set to now
       .setExpirationTime("1h")
```

Comment:
```suggestion
      .setExpirationTime("5min")
```

Reply:
Fixed in 5621aeba7e2bece3d95a5060353a4c72e7eec56b, thanks.